cherrypy/test/test_bus.py

```diff
@@ -9,7 +11,7 @@
 
 from cherrypy.process import wspbus
 
-
+CI_ON_MACOS = bool(os.getenv("CI")) and sys.platform == "darwin"
```

Comment:
Preserve the "two lines after imports" spacing
```suggestion

CI_ON_MACOS = bool(os.getenv("CI")) and sys.platform == "darwin"
```

Reply:
Can we please format the codebase with [psf/black](https://github.com/psf/black) and eliminate the arbitrary code formatting rules?

Reply:
No, black is so terrible that I avoid projects pushing it down the throat.
Also, it's not arbitrary. Common sense tells us not to change the formatting unrelated to other changes in the PR so why do that at all? I'm not asking for anything that doesn't make sense: there's a visual separation that's already maintained there — just don't break it, I don't ask to reformat anything in the actual changes. Also, none of the existing formatters is clever enough to keep the visual separators meant for humans, not machines. If anything's not right beside what pre-commit reports, I'd rather fix it myself than annoy the contributors.
Besides, there's enough autoformatters in our pre-commit setup: https://travis-ci.org/github/cherrypy/cherrypy/jobs/743940552#L291.